source/site-administrators/studio/bucket-storage-publishing.rst

```diff
@@ -0,0 +1,178 @@
+
+  .. code-block:: xml
+     :emphasize-lines: 2
+
+     <published-repository>
+         <enable-staging-environment>true</enable-staging-environment>
+         <staging-environment>staging</staging-environment>
+         <live-environment>live</live-environment>
+     </published-repository>
+
+  |
+
+For more information on staging, see :ref:`staging-env`
+
+^^^^^^^^^^^^^^^^^^^
+2. Setup Blob Store
+^^^^^^^^^^^^^^^^^^^
+
+In your Studio, click on |siteConfig| -> *Configuration* -> *Blob Stores* and fill in the required information to setup the S3 buckets for the preview, staging and live.
+
+   .. code-block:: xml
+      :linenos:
+      :emphasize-lines: 9,14,19,24,25,27
+
+      <blobStores>
+        <blobStore>
+          <id>s3-default</id>
+          <type>s3BlobStore</type>
+          <pattern>/static-assets/s3/.*</pattern>
+          <mappings>
+            <mapping>
+              <publishingTarget>preview</publishingTarget>
+              <storeTarget>my-authoring-bucket</storeTarget>
+              <prefix>sandbox</prefix>
+            </mapping>
+            <mapping>
+              <publishingTarget>staging</publishingTarget>
+              <storeTarget>my-authoring-bucket</storeTarget>
+              <prefix>staging</prefix>
+            </mapping>
+            <mapping>
+              <publishingTarget>live</publishingTarget>
+              <storeTarget>my-delivery-bucket</storeTarget>
+            </mapping>
+          </mappings>
+          <configuration>
+            <credentials>
+              <accessKey>xxxxxxxxx</accessKey>
+              <secretKey>xxxxxxxxx</secretKey>
+            </credentials>
+            <region>us-west-1</region>
+            <pathStyleAccess>true</pathStyleAccess>
+          </configuration>
+        </blobStore>
+      </blobStores>
+
+   |
+
+To see more information on the Blob Stores configuration, see :ref:`blob-stores-configuration`
+
+^^^^^^^^^^^^^^^^^^^
+3. Create s3 folder
+^^^^^^^^^^^^^^^^^^^
+
+Create an ``s3`` folder under ``static-assets``.  This folder is automatically  mapped to the AWS S3 buckets we setup in the previous step.
+To create the folder, open the ``Sidebar`` and navigate to ``static-assets``.  Right click on the ``static-assets`` folder, then select ``Create Folder``.  Enter ``s3`` as folder name then click ``OK``
+
+.. image:: /_static/images/site-admin/bucket/create-s3-folder.png
+    :align: center
+    :alt: "s3" folder created under "static-assets"
+    :width: 35%
+
+|
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+4. Upload files to bucket storage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To upload files to the S3 buckets we setup earlier, open the ``Sidebar`` then navigate to ``static-assets`` -> ``s3``.  Create a folder named ``docs`` under ``s3``.  Right click on the newly created folder and select ``Upload`` to upload a single file, or ``Bulk Upload`` to upload multiple files
```

Comment:
@alhambrav I think we should also mention other uploads options, for example the upload data sources and maybe even include an example?